net/wol/src/opnsense/www/js/widgets/WakeOnLan.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2024 Michał Brzeziński
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ * INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ * AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ * OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import BaseTableWidget from "./BaseTableWidget.js";
```

Comment:
that's the old old style. I think for now we can remove this and I'll fix a backport if we need it.

```suggestion
```

Reply:
Without this import, on the latest stable (24.7.1) I have this error:

```
[ReferenceError: BaseTableWidget is not defined
    at https://opn-m3e/ui/js/widgets/WakeOnLan.js?t=…]
```
![image](https://github.com/user-attachments/assets/1d64f085-991f-43ab-ac13-0bbfa290658e)

Reply:
21.7.1 & .2 actually require an importmap-based statement. 24.7 required your current statement. 24.7.3 will likely require no statement which is also the version we need to merge to the master branch

Reply:
Thanks for the explanation. I've committed the version without the import statement